src/Ortho.jl

```diff
@@ -78,20 +78,20 @@ function ortho2norm(t::CharTable, class::Int64)
 end
 
 @doc raw"""
-    ortho2norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly
+    norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly
 
 Return the (generic) norm of the class type `class`.
 
 # Examples
 ```jldoctest
 julia> g=greenfuntab("GL3");
 
-julia> ortho2norm(g,2)
+julia> norm(g,2)
 (5*q^2 + 2*q + 3)//(q^5 - 2*q^4 + q^3)
 ```
 """
-function ortho2norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly  # TODO is correct?
-	if class > classtypes(t)
+function Oscar.norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly  # TODO is correct?
```

Comment:
We should just `import Oscar: norm` in a suitable place and then do
```suggestion
function norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly  # TODO is correct?
```